src/corelibs/U2Gui/src/util/SaveDocumentController.cpp

```diff
@@ -143,6 +142,17 @@ DocumentFormatId SaveDocumentController::getFormatIdToSave() const {
     return formatsInfo.getIdByName(currentFormat);
 }
 
+QString SaveDocumentController::getValidatedSaveFilePath(U2OpStatus &os) const {
+    QString fileName = getSaveFileName();
+    CHECK_EXT(!fileName.isEmpty(), os.setError(tr("Output file name is empty")), "");
+
+    CHECK_EXT(FileAndDirectoryUtils::canWriteToPath(fileName),
+              os.setError(tr("File location is not writable: %1").arg(fileName)),
```

Comment:
You could use `L10N::errorOpeningFileWrite`

Reply:
Fixed